SimpleRecorder/SimpleRecorder.csproj

```diff
@@ -22,7 +22,6 @@
     <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
     <AppxAutoIncrementPackageRevision>True</AppxAutoIncrementPackageRevision>
     <AppxBundlePlatforms>x86|x64|arm|arm64</AppxBundlePlatforms>
-    <PackageCertificateThumbprint>2C29998D36211BB933BF95E05723B5154D0559A9</PackageCertificateThumbprint>
```

Comment:
This broke the build if included and I think has something to do with your machine local cert store? so probably not worth checking in